### ISSUE TITLE
Please include build folder for npm package

### ISSUE THREAD

**Issue by RusinovAnton** [derhuerst/window-on-print#2]
Hello! Nice looking package. But I have probs to compile it for my production, because build folder isn't there and my UglifyJs is failing on ES6 syntax. Thanks.

**Comment by derhuerst** [comment]
Will address this soon, thanks!

**Comment by RusinovAnton** [comment]
Also I didn't understand why it unsubscribe itself on event trigger?

Look how I used your code for ReactJS high order component.
```js
import React from 'react'


// window-on-print
// https://github.com/derhuerst/window-on-print
const onPrint = (type, match) => (cb) => {
  let query;
  let queryFn;

  if (window.matchMedia) {
    queryFn = (query) => {
      if (query.matches === match) cb()
    }
    query = window.matchMedia('print')
    query.addListener(queryFn)
  } else {
    window.addEventListener(type + 'print', cb, false)
  }

  return function unsubscribe() {
    if (queryFn) query.removeListener(queryFn)
    window.removeEventListener(type + 'print', cb, false)
  }
}

const beforePrint = onPrint('before', true);
const afterPrint = onPrint('after', false);


const withPrintState = (Component) => {
  class WrappedWithPrintState extends React.PureComponent {
    state = {
      printState: false
    }

    componentDidMount() {
      this.unsubscribeBefore = beforePrint(() => {
        this.setState({ printState: true });
      });

      this.unsubscribeAfter = afterPrint(() => {
        this.setState({ printState: false });
      });
    }

    componentWillUnmount() {
      this.unsubscribeBefore();
      this.unsubscribeAfter();
    }

    render() {
      return <Component {...this.props} isPrint={this.state.printState}/>
    }
  }

  return WrappedWithPrintState
}

export default withPrintState
```

**Comment by derhuerst** [comment]
> Also I didn't understand why it unsubscribe itself on event trigger?

You solution to return the `unsubscribe` fn looks nice! Will implement it.

**Comment by derhuerst** [comment]
[The build is failing](https://travis-ci.org/derhuerst/window-on-print/builds/259477424) 😢. Will deal with it later.

**Comment by derhuerst** [comment]
Published `window-on-print@0.2.0`, which should contain both the `unsubscribe` API proposed by you as well as transpiled source code in `dist`.

Please reopen if you encounter any issues with it.
